app/controllers/admin/tasks_controller.rb

```diff
@@ -42,4 +52,11 @@ def next_task_path
       new_admin_claim_decision_path(@claim)
     end
   end
+
+  def check_params
+    params.fetch(:task, {})
```

Comment:
Because we always expect the params to arrive via a scoped `task` form we would be better off using the more standard `params.require(:task)` line here. We may need to add an `f.hidden_field :passed` line to the form to ensure we always get something posted, but that's a standard pattern that we're already employing in the yes/no forms in the public-facing site.

Reply:
Ah yeah, I put that in initally to keep the tests green, but forgot to take it out

Reply:
This is still using `fetch`, were you going to change it to `require`?

Reply:
Arrrgh, dammit. Yep

Reply:
Sorted, also added another spec to make sure it works with the hidden field in play